src/types/react.ts

```diff
@@ -0,0 +1,6 @@
+import { AriaAttributes, DOMAttributes } from 'react'
+
+export type ReactHtmlElementProps<T extends AriaAttributes | DOMAttributes<T>>
+  = React.DetailedHTMLProps<React.HTMLAttributes<T>, T>
+
+export type DefaultComponentProps = Pick<ReactHtmlElementProps<HTMLDivElement>, 'className' | 'style'>
```

Comment:
So this is the only way to implement this right? Not sure if this works for all our components since not all of them are div-elements :D

Reply:
Mmmh this `DefaultComponentProps` type only contain the exact React's type for `className` and `style`. Theses props are the same for all HTML-elements so it's not a problem to just use `HTMLDivElement` in this case.